src/v/kafka/server/handlers/create_topics.cc

```diff
@@ -43,7 +43,8 @@ static constexpr std::array<std::string_view, 10> supported_configs{
    "retention.ms",
    "redpanda.remote.recovery",
    "redpanda.remote.write",
-   "redpanda.remote.read"}};
+   "redpanda.remote.read",
+   "redpanda.read.replica"}};
```

Comment:
what about the bucket name?

Reply:
@LenaAn why don't we do  `if property starts_with("redpanda.") is good` kinda thing.

Reply:
you mean instead of an array of supported configs? But not all configs that start with "redpanda." are supported